### DISCUSSION TITLE
How to manually specify path to config.json?

### DISCUSSION THREAD

**Discussion by slimy-pufflefish** [bghira/SimpleTuner#1173] [Q&A] [answered]
How can I specify the path to the config.json on the CLI? I see there's a `train.sh` script, but it doesn't look like it accepts a path to the config.json file.

**Comment by bghira** [comment, marked as answer]
it is always in the `config/` dir, or if you use `ENV=foobar ./train.sh` then it is in `config/foobar/config.json`
